### PR TITLE
My first branch

### PR DESCRIPTION
**Pull request by Faret23**

### CODE REVIEW

**Comment by Faret23** [comment]
homework 30-11-2020

File: stadium.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+stadium::stadium(string a, int b, int c, string d, int e, string f, int g) {
+	name = a;
+	number_of_spectators = b;
+	lighting_power = c;
+	city = d;
+	ticket_price = e;
+	country = f;
+	square = g;
```
Comment (TarasBek) [inline stadium.cpp:37]: What is square? Please rename it)

File: stadium.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	string city;
+	int ticket_price;
+public:
+	string country;
+	int square;
+
+	int getNumber();
+	string getName();
+	int getPower();
+	void getResult();
```
Comment (TarasBek) [inline stadium.h:22]: I don't think you need string, int, void before these methods. Please, delete them)

**Review by TarasBek — CHANGES_REQUESTED** [review]

File: ConsoleApplication4.cpp [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+int main() {
+	stadium A("Arena Lviv", 34915, 2400, "Lviv", 150, "Ukraine", 7140);
+	stadium B("Chornomorets", 34164, 1700, "Odessa", 200, "Ukraine", 7140);
+	stadium C("Dnipro-Arena", 31003, 1800, "Dnipro", 175, "Ukraine", 7140);
+	A.getResult();
+	B.getResult();
+	C.getResult();
+	return 0;
+}
```
[the remark this answers is not in the document]
Comment (TarasBek) [reply ConsoleApplication4.cpp:12]: > please add a new line at the end of the main function

**Pull request merged**